README.md

```diff
@@ -1 +1,189 @@
-# first-capstone
\ No newline at end of file
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 \[interARTction\] ](#-interartction-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+    - [Install](#install)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ (OPTIONAL) ](#-faq-optional-)
+  - [📝 License ](#-license-)
+  
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [interARTction] <a name="about-project"></a>
+
+[interARTction] is a website which is for the Interdisciplinary Art Exhibition which will be held in Yangon, Myanmar.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.w3schools.com/html/">HTML</a></li>
+     <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+     <li><a href="https://www.w3schools.com/css/">Javascript</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Html structure with-
+  - Heading and Section
+  - Introduction
+  - Social media icons
+- CSS styling with -
+  - Color
+  - Font Style
+  - Hover effects
+  - FLex box and Grid
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://pi1998.github.io/first-capstone/)
```

Comment:
- [ Required ] I think it would be better if you deployed your project online as requested by the previous reviewer, you could deploy your project using GitHub Pages. Deploying your project online is good especially if you want recruiters and clients to have a look at some of the projects you've worked on. I would like to suggest this article on how to deploy a project using [GitHub Pages publishing from a branch >>](https://docs.github.com/en/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site#publishing-from-a-branch)

Reply:
> - [ ] Good job @Pi1998, But I noticed that on your deployed website, when I try to navigate to the home page from the About page, the page link is broken and I get a 404 error😞It seems there is no home.html in your root directory which it is pointing to. Please could you kindly fix that? You could point the home nav link to index.html instead of home.html. This will ensure a better user experience for your users/visitors.😊 
> 
> Your Home Page👇:
> 
> ![home](https://github.com/Pi1998/first-capstone/assets/40009354/000ac445-b087-41e7-ae62-2e47e1706e1e)